src/os/file_anyos_test.go

```diff
@@ -0,0 +1,25 @@
+package os_test
+
+import (
+	. "os"
+	"testing"
+)
+
+// TestTempDir assumes that the test environment has a filesystem with a working temp dir.
+func TestTempDir(t *testing.T) {
+	name := TempDir() + "/_os_test_TestTempDir"
+	Remove(name)
+	f, err := OpenFile(name, O_RDWR|O_CREATE, 0644)
+	if err != nil {
+		t.Errorf("OpenFile %s: %s", name, err)
+		return
```

Comment:
Suggestion: I believe this can be replaced with `t.Fatal`.

Reply:
It would still need the return at the moment, I went for lower cognitive dissonance :-)

Reply:
Fair enough. Indeed, `t.Fatal` isn't obviously returning (even though the effect is similar).